test/templates/bar/index.njk

```diff
@@ -0,0 +1,500 @@
+{% extends "layout.njk" %}
+{% from "./../format.njk" import format %}
+
+{% block content %}
+
+
+<h2>Bar - Desktop</h2>
```

Comment:
Since we noticed how much number of test images reflects on pipe performance, could we somehow group our test cases? Like Header + all modification cases, Subheader + all modification cases etc.

Reply:
Will do this. Makes sense